--- "/home/griggorii/20.04/lesspipe"	2019-09-03 12:41:52.000000000 +0800
+++ "/home/griggorii/21.04/lesspipe"	2020-07-06 04:01:10.000000000 +0800
@@ -48,10 +48,10 @@
 	# generate filename for possible use by lesspipe
 	umask 077
 	if [ $BASENAME = $LESSFILE ]; then
-		TMPFILE=`tempfile -d $TMPDIR -p lessf`
+		TMPFILE=`mktemp -p $TMPDIR lessfXXXXXX`
 		if [ -z "$TMPFILE" ]; then
-			echo >&2 "Could not find essential program 'tempfile'. Exiting"
-      exit 1
+			echo >&2 "TMPFILE variable is empty. Exiting"
+		exit 1
 		fi
 	fi
 

                                                        Griggorii@gmail.com       

-		TMPFILE=`tempfile -d $TMPDIR -p lessf`          <-ubuntu-special-os-20.04
+		TMPFILE=`mktemp -p $TMPDIR lessfXXXXXX`  <- ubuntu-21.04-zsync
